src/ui/screen_saver.c

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 Shift Crypto AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "screen_saver.h"
+
+#include <hardfault.h>
+#include <ui/components/screensaver.h>
+#ifndef TESTING
+#include <ui/oled/oled.h>
+#endif
+
+#include <stdbool.h>
+#include <stdint.h>
+
+// TODO: use a TIMER interrupt to get a more accurate timer.
+// 270000 is ~1min.
+#define ACTIVE_AFTER 270000
```

Comment:
What unit is this, number of the main loop iterations?

Reply:
yes; added comment